repo/passwordless.js

```diff
@@ -0,0 +1,49 @@
+const error = require('error')
+const {db} = require('db')
+const {mapper} = require('repo/base')
+
+const map = mapper({
+  tokenRemote: 'token_remote',
+  tokenDirect: 'token_direct',
+  userId: 'user_id',
+  createdAt: 'created_at',
+})
+
+async function create (tokenRemote, tokenDirect, userId) {
+  return db.none(`
```

Comment:
Here `await` instead of `return` would probably be more clear (In sync code, the return would be unnecessary.)